### ISSUE TITLE
how to correctly do classifying?

### ISSUE THREAD

**Issue by junchen14** [huggingface/pytorch-pretrained-BERT#470]
when I do the classifying with task cola, the result is like this?
eval_loss = 0.0
global_step = 49173
loss = 0.0
mcc = 0.0

what is my prediction result?
and how should I use the output model????

**Comment by jtfields** [comment]
I'm having a similar issue where I run a BERT tutorial  by Chris McCormick and receive 65-78% accuracy on training data and 0.0% for test data.  Was anyone able to diagnose a result of 0.0%?  One theory I have is a division by zero error for very small numbers that are rounded to zero...
